src/rendering/screenspacerenderable.cpp

```diff
@@ -522,18 +533,60 @@ glm::mat4 ScreenSpaceRenderable::scaleMatrix() {
 
     glm::mat4 scale = glm::scale(
         glm::mat4(1.f),
-        glm::vec3(_scale, _scale * textureRatio, 1.f)
+        glm::vec3(_scale, textureRatio*_scale, 1.f)
     );
+  
+    return scale;
+}
+glm::vec2  ScreenSpaceRenderable::screenSpacePosition() {
```

Comment:
please add a new line before the function, and remove the extra space after `glm::vec2`